src/server/routes/index.ts

```diff
@@ -41,6 +42,9 @@ uncachedRoutes.use(verifyEmail);
 // magic link routes
 uncachedRoutes.use(magicLink);
```

Comment:
Unrelated, but should this route also use `noCache`?

Reply:
it does already! `uncachedRoutes` is a wrapper which Peter created in #788 to automatically apply `noCache`